CHANGELOG.rst

```diff
@@ -1,8 +1,12 @@
-DEV
+0.6.0
 ------------------
+* Remove `six` dependency
+* Remove `User.save` method
```

Comment:
Why? This is expected for modification, isn't it?

Reply:
It's used only for creation. Also for creation, we use additional arguments `is_ws` and `site_url`. So for using `User.save` method we need to add `is_ws` and `site_url` for `User._fields`, but for getting user, no `is_ws` in it and `site_url` in `User.preferences`. So was added separate classmethod for creation